sxapi/low.py

```diff
@@ -73,7 +74,10 @@ def stats(self):
         return out
 
     def to_url(self, path, version_modifier=None):
-        url = "{}{}".format(self.api_base_url, path)
+        base_url = self.api_base_url
+        if path.startswith('/data'):
+            base_url = PUBLIC_API_V2.rstrip("/")
+        url = "{}{}".format(base_url, path)
```

Comment:
You can write this change as
url = "{}{}".format(self.api_base_url, path)

Reply:
but then I would no change the api version for data calls, since self.api_base_url is public v1

Reply:
You are right I did not see that the base url is set to the old api at the start :)

Reply:
I think that the API which should be used should be a parameter in the function call.

Reply:
why didn't you use the version_modifier parameter?

Reply:
I think Philipp is right: we should try to find a way to use the version modifier to go to the correct version of the API.